### ISSUE TITLE
Flurry stoped building build for iOS

### ISSUE THREAD

**Issue by maheshk27** [flurry/flutter-flurry-sdk#12]
Unable to build the iOS build with flurry.

Following is the error details from terminal. Followed the steps as mentioned in the developer doc for the linking the library

https://developer.yahoo.com/flurry/docs/integrateflurry/ios/#swift

```
Xcode's output:
↳
    Writing result bundle at path:
    	/var/folders/sm/0cmg_3td2_3d03jz5r7mhd5r0000gn/T/flutter_tools.uPtECS/flutter_ios_build_temp_dirrzOfNZ/temporary_xcresult_bundle

    /Users/maheshkarande/.pub-cache/hosted/pub.dartlang.org/flutter_flurry_sdk-2.0.0/ios/Classes/FlurryFlutterPlugin.m:428:39: warning: implicit conversion loses integer precision: 'NSInteger' (aka 'long') to
    'int' [-Wshorten-64-to-32]
        [Flurry setSessionContinueSeconds:secondsInt];
        ~                                 ^~~~~~~~~~
    1 warning generated.
    /Users/maheshkarande/.pub-cache/hosted/pub.dartlang.org/flutter_flurry_sdk-2.0.0/ios/Classes/FlurryFlutterPlugin.m:428:39: warning: implicit conversion loses integer precision: 'NSInteger' (aka 'long') to
    'int' [-Wshorten-64-to-32]
        [Flurry setSessionContinueSeconds:secondsInt];
        ~                                 ^~~~~~~~~~
    1 warning generated.
    Command CompileSwiftSources failed with a nonzero exit code
    remark: Incremental compilation has been disabled: it is not compatible with whole module optimization
    Command CompileSwiftSources failed with a nonzero exit code
    Undefined symbols for architecture x86_64:
      "_OBJC_CLASS_$_FlurryParamBuilder", referenced from:
          objc-class-ref in FlurryFlutterPlugin.o
      "_OBJC_CLASS_$_FlurryCCPA", referenced from:
          objc-class-ref in FlurryFlutterPlugin.o
      "_OBJC_CLASS_$_FlurryUserProperties", referenced from:
          objc-class-ref in FlurryFlutterPlugin.o
      "_OBJC_CLASS_$_Flurry", referenced from:
          objc-class-ref in FlurryFlutterPlugin.o
         (maybe you meant: _OBJC_CLASS_$_FlurryFlutterPlugin)
      "_OBJC_CLASS_$_FlurrySessionBuilder", referenced from:
          objc-class-ref in FlurryFlutterPlugin.o
      "_OBJC_CLASS_$_FConfig", referenced from:
          objc-class-ref in FlurryFlutterPlugin.o
      "_OBJC_CLASS_$_FlurryMessaging", referenced from:
          objc-class-ref in FlurryFlutterPlugin.o
    ld: symbol(s) not found for architecture x86_64
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    note: Using new build system
    note: Planning
    note: Build preparation complete
    note: Building targets in dependency order
 ```
 
POD File

```
# Uncomment this line to define a global platform for your project
# platform :ios, '9.0'

# CocoaPods analytics sends network stats synchronously affecting flutter build latency.
ENV['COCOAPODS_DISABLE_STATS'] = 'true'

project 'Runner', {
  'Debug' => :debug,
  'Profile' => :release,
  'Release' => :release,
}

def flutter_root
  generated_xcode_build_settings_path = File.expand_path(File.join('..', 'Flutter', 'Generated.xcconfig'), __FILE__)
  unless File.exist?(generated_xcode_build_settings_path)
    raise "#{generated_xcode_build_settings_path} must exist. If you're running pod install manually, make sure flutter pub get is executed first"
  end

  File.foreach(generated_xcode_build_settings_path) do |line|
    matches = line.match(/FLUTTER_ROOT\=(.*)/)
    return matches[1].strip if matches
  end
  raise "FLUTTER_ROOT not found in #{generated_xcode_build_settings_path}. Try deleting Generated.xcconfig, then run flutter pub get"
end

require File.expand_path(File.join('packages', 'flutter_tools', 'bin', 'podhelper'), flutter_root)

flutter_ios_podfile_setup

target 'Runner' do
  use_frameworks!
  use_modular_headers!

  pod 'Flurry-iOS-SDK/FlurrySDK' #Analytics Pod
  # pod 'Flurry-iOS-SDK/FlurryAds' #Advertising Pod (requires Analytics)

  flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
end

pre_install do |installer|
  # workaround for https://github.com/CocoaPods/CocoaPods/issues/3289
  Pod::Installer::Xcode::TargetValidator.send(:define_method, :verify_no_static_framework_transitive_dependencies) {}
end

post_install do |installer|
  installer.pods_project.targets.each do |target|
    flutter_additional_ios_build_settings(target)
  end
end
```

**Comment by randompaga** [comment]
The same error

**Comment by hantao-flurrydev** [comment]
Hi @maheshk27 @randompaga, thanks for reporting this issue. We recently released 12.0.0 for native iOS SDK, and looks like the flutter plugin integrated the latest native iOS sdk since we don't specify the version in `.podspec` file. We will release a patch shortly to fix the missing arm64 issue in native sdk, and will update you once release is done.

**Comment by longma-tech** [comment]
So long time..... when will you fix this bug??

**Comment by poting-oath** [comment]
@maheshk27 @randompaga @longma-tech This crash is caused by the native Flurry SDK (our plugin is just a wrapper). We were waiting for the new native SDK release which was just released one week ago. We are going to release our plugin soon. Thanks for your patient!

**Comment by poting-oath** [comment]
Fix released at flutter_flurry_sdk 2.2.0.

**Comment by poting-oath** [comment]
@maheshk27 @randompaga @longma-tech More Fix released at flutter_flurry_sdk 2.2.1.
